functorch/_src/aot_autograd.py

```diff
@@ -865,7 +868,18 @@ def forward(self, *args, **kwargs):
     return AOTModule()
 
 
-def aot_module_simplified(mod: nn.Module, *top_args, **top_kwargs) -> nn.Module:
+def aot_module_simplified(
+        mod: nn.Module,
+        inputs,
+        fw_compiler: Callable,
+        bw_compiler: Optional[Callable] = None,
+        partition_fn: Callable = default_partition,
+        decompositions: Optional[Dict] = None,
+        hasher_type=None,
+        static_argnums=None,
+        fake_mode=None,
```

Comment:
What the heck is this kwarg, you ignore it immediately after?

Reply:
Leftover